### ISSUE TITLE
Document breaking changes in the update pipeline

### ISSUE THREAD

**Issue by roji** [dotnet/EntityFramework.Docs#3769]
Issues:

* [ ] https://github.com/dotnet/efcore/issues/27439
* [ ] https://github.com/dotnet/efcore/issues/27372
* [ ] https://github.com/dotnet/efcore/issues/27532
* [ ] https://github.com/dotnet/efcore/issues/27547

Since the breaking changes here are quite wide-reaching, reach out proactively to provider writers to help them.

**Comment by ajcvickers** [comment]
@roji I didn't work on this when doing the 7.0 breaking changes because I'm not sure what needs doing where.

**Comment by roji** [comment]
Yeah, to be honest, I'm not sure what user-facing breaking changes we have here (apart from the trigger trouble which is already documented). The fact we no longer wrap single statements in a transaction *may* theoretically cause a behavior change in some scenarios, but I'd have to think hard to find such a scenario (we should maybe wait until a user hits one?).

(There are certainly many provider-facing breaking changes here, but I'm not a fan of doing a breaking change note for those)

**Comment by ajcvickers** [comment]
Agreed.

**Comment by ErikEJ** [comment]
Maybe that EF Core runs too fast now? 🤣

**Comment by roji** [comment]
@ErikEJ something like [this](https://xkcd.com/1172/)...

**Comment by ErikEJ** [comment]
Exactly. Luckily you have a "go slow" switch.
